software/bedtools/genomecov/main.nf

```diff
@@ -0,0 +1,34 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+def options    = initOptions(params.options)
+
+process BEDTOOLS_GENOMECOV {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::bedtools =2.29.2" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/bedtools:2.29.2--hc088bd4_0"
+    } else {
+    container "quay.io/biocontainers/bedtools:2.29.2--hc088bd4_0"
+    }
+
+    input:
+        tuple val(meta), path(bams), path (sizes)
+
+    output:
+        tuple val(meta), path("*.bed"), emit: coverage
+        path  "*.version.txt", emit: version
+
+    script:
+        def software = getSoftwareName(task.process)
+        def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+        """
+        bedtools genomecov -ibam $bams -g $sizes ${options.args} > ${prefix}.bed
```

Comment:
You shouldn't need a sizes file with BAM input. I think you will get a warning in the log output from BEDTools. See [here](https://github.com/drpatelh/nf-core-viralrecon/blob/24ca7bd00600a30fb75c1e0774841d86055f465d/modules/local/bedtools_genomecov.nf#L32-L36)

Can this command take multiple `bams` as input?

Be great if we indent the command as in the above module too as well as in the `input` and `output` and `script` sections. I have added corrections for the `complement` module to give you an idea.

Reply:
I just saw that error! Thank you for letting me know - I fixed that. It doesn't look like multiple bams can be taken as an input based on the documentation.
I went ahead and updated all of the modules to be indented/follow the current nf-core syntax (checked to see if they ran as well)- I will let you know once they are ready to be merged!